### PR TITLE
Centered the footer, added a gradient background for the footer. Changes made in index.css and index.html

### PR DESCRIPTION
**Pull request by sibayanmisra2002**
image

Fixed the footer CSS and added Copyrights mark. Finally with a background it's looking good.

### CODE REVIEW

**Review by arpitghura — CHANGES_REQUESTED** [review]
Made changes as per suggestions.

**Comment by sibayanmisra2002** [comment]
> Made changes as per suggestions.

Done, please check the last commit in index.css

File: index.css [hunk 1]
```diff
@@ -152,6 +158,10 @@ footer p{
     .brand{
         margin: auto;
     }
+    footer{
+        justify-content: center;
```
Comment (arpitghura) [inline index.css:162]: duplicate use of this property, already defined in the footer tag. 
Remove it.

**Review by arpitghura — CHANGES_REQUESTED** [review]
Remove the tag in media queries, check review

**Review by arpitghura — APPROVED** [review]
Okay Working Fine

**Pull request merged**